scripts/marc_comparison/match_up_file.py

```diff
@@ -199,10 +200,6 @@ def marc_compare(file1, file2, idloc1, idloc2):
             num_records_with_differences += 1
 
         differences = nested_counter_update(differences, new_diffs)
-
-    # Print summary of differences
-    if num_records_with_differences > 0:
```

Comment:
I'm assuming this isn't important anymore.

Reply:
Yeah, that was just an extra newline at the end of the report which is taken care of by the other change.